DOC-1341: RS - Update LDAP docs to reflect changes (#1936)
RedisLabs/redislabs-docs: content/rs/security/ldap/migrate-to-role-based-ldap.md

aliases: 
---

Redis Enterprise Software supports LDAP through a [role-based mechanism]({{< relref "/rs/security/ldap/" >}}), first introduced [in v6.0.20]({{< relref "rs/release-notes/rs-6-0-20-april-2021" >}}).

> (Support for a earlier, [cluster-based mechanism]({{< relref "/rs/security/ldap/cluster-based-ldap-authentication.md" >}}) was removed in v6.2.12.)

Reviewer: ```suggestion
(Support for an earlier, [cluster-based mechanism]({{< relref "/rs/security/ldap/cluster-based-ldap-authentication.md" >}}) was removed in v6.2.12.)
```

Author: Fixed.

 Because deployments and requirements vary, you’ll likely need to adjust these guidelines.

## Test LDAP access

> To test your LDAP integration, including:

Reviewer: ```suggestion
To test your LDAP integration, you can:
```

> - Connecting with `redis-cli` and using the AUTH command to test LDAP username/password credentials.

Reviewer: Maybe remove the -ing verbs in this list of test methods.
```suggestion
- Connect with `redis-cli` and use the `AUTH` command to test LDAP username/password credentials.
```

Author: Fixed.
